apps/organisations/views.py

```diff
@@ -215,44 +218,93 @@ def get_context_data(self, **kwargs):
+        image_get = Image.open(data['image'].file)
+        image_size = (sharepic_format['img_min_width'],
+                      sharepic_format['img_min_height'])
+        image = image_get.resize(image_size)
+
+        # Get required total size and add appropriate padding
         right = 0
         left = 0
-        top = 64
-        bottom = 48
+        top = 0
+        bottom = (
+            sharepic_format['overall_height'] -
+            sharepic_format['img_min_height'])
         width, height = image.size
         new_width = width + right + left
         new_height = height + top + bottom
         result = Image.new(
             image.mode, (new_width, new_height), (255, 255, 255))
         result.paste(image, (left, top))
 
-        # Adding writting
-        # Image is converted into editable form using Draw function and
-        # assigned to d1
-        d1 = ImageDraw.Draw(result)
-        # Text location, color and font
+        # image is converted into editable form using Draw function
+        image = ImageDraw.Draw(result)
+
+        # text dimension, color and font
         font = ImageFont.truetype(
-            "adhocracy-plus/assets/fonts/SourceSansPro-Semibold.otf", 20)
+            "adhocracy-plus/assets/fonts/SourceSansPro-Semibold.otf",
+            sharepic_format['title_size'])
         fontsm = ImageFont.truetype(
-            "adhocracy-plus/assets/fonts/SourceSansPro-Regular.otf", 12)
-        d1.text((16, 12), data['title'], fill=(0, 0, 0), font=font)
-        d1.text((16, 37), data['description'], fill=(0, 0, 0), font=fontsm)
-
-        # Adding logos
-        result.paste(logo1, (46, 277))
-        result.paste(logo2, (157, 277))
+            "adhocracy-plus/assets/fonts/SourceSansPro-Regular.otf",
+            sharepic_format['description_size'])
+        title = data['title']
+        description = data['description']
+        title_width, title_height = image.textsize(title, font=font)
+        description_width, description_height = image.textsize(
+            description, font=fontsm)
+
+        # add text using width to center with 20px padding
+        image.text(((new_width - title_width) / 2,
+                    (sharepic_format['img_min_height'] + 20)),
+                   data['title'], fill=(0, 0, 0), font=font)
+        image.text(
+            ((new_width - description_width) / 2,
+             (sharepic_format['img_min_height'] +
+                sharepic_format['title_size'] + 40)),
+            data['description'],
+            fill=(0, 0, 0),
+            font=fontsm)
+
+        if data['add_orga_logo']:
+            # Get organisations logo
+            logo_org_get = Image.open(self.organisation.logo)
+            logo_org_size = (144, 144)
+            logo_org = logo_org_get.resize(logo_org_size)
+            # position org logo
+            result.paste(logo_org, (160, 160))
+
+        if data['add_aplus_logo']:
+            # Get aplus logo
+            logo_aplus_get = Image.open(
```

Comment:
i think we should maybe load this from the assets folder, otherwise a fresh dev environment won't work out of the box with our Makefile.
One way could be:
```
logo_aplus_get = Image.open(
                os.path.join(
                    settings.BASE_DIR,
                    'adhocracy-plus/assets/images/logo.png'))
```
What do you think @Rineee @fuzzylogic2000

Reply:
Yes, I think that would work!